packages/web-app-files/src/search/sdk/list.ts

```diff
@@ -0,0 +1,29 @@
+import { SearchList, SearchResult } from 'search/src/types'
+import ListComponent from '../../components/Search/List.vue'
+import { clientService } from '../../services'
+import { buildResource } from '../../helpers/resources'
+import { Store } from 'vuex'
+import { Component } from 'vue'
+
+export default class List implements SearchList {
+  public readonly component: Component
+  private readonly store: Store<any>
+
+  constructor(store: Store<any>) {
+    this.component = ListComponent
+    this.store = store
+  }
+
+  async search(term: string): Promise<SearchResult[]> {
+    const plainResources = await clientService.owncloudSdk.files.search(
+      term,
+      undefined, // add configuration option, other places need that too too... needs consolidation
```

Comment:
What to do here, keep the comment?

Reply:
i removed it for now, needs to be done once we have a solid configuration management in place